artifacts/e2e-failure-report.md

```
@@ -0,0 +1,187 @@
+
+
+---
+
+## Root Cause Classifications &amp; Evidence
+
+### 1. `tests/e2e/completed-worker-retirement-resume.spec.ts:39:3`
+
+- **Job**: `e2e 1-of-10` (Job ID: `97985466208`)
+- **Classification**: `TEST_UPDATE`
+- **Error**:
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)